docs/pipeline.rst

```diff
@@ -4,42 +4,45 @@ Pipeline
 Pipeline class
 --------------
 
-.. currentmodule:: nvidia.dali.pipeline
+.. currentmodule:: nvidia.dali
 
-In DALI, any data processing task has a central object called pipeline. Pipeline object is an
-instance of :class:`nvidia.dali.pipeline.Pipeline` or a derived class. Pipeline encapsulates the
+In DALI, any data processing task has a central object called Pipeline. Pipeline object is an
+instance of :class:`nvidia.dali.Pipeline` or a derived class. Pipeline encapsulates the
 data processing graph and the execution engine.
 
-There are two ways to define a DALI pipelines:
+You can define a DALI Pipeline in the following ways:
 
-#. by inheriting from Pipeline class and overriding :meth:`Pipeline.define_graph`
-#. by instantiating `Pipeline` directly, building the graph and setting the pipeline
-   outputs with :meth:`Pipeline.set_outputs`
+#. by implementing a function that uses DALI's ``Operators`` inside and decorating it with the
+:meth:`pipeline` decorator
+#. by instantiating :class:`Pipeline` object directly, building the graph and setting the pipeline
+outputs with :meth:`Pipeline.set_outputs`
+#. by inheriting from :class:`Pipeline` class and overriding :meth:`Pipeline.define_graph`
+(this is the legacy way of defining DALI Pipelines)
 
 Data processing graphs
 """"""""""""""""""""""
 
 DALI pipeline is represented as a graph of operations. There are two kinds of nodes in the graph:
 
  * Operators - created on each call to an operator
- * Data nodes (see :class:`DataNode`) - represent outputs and inputs of operators; they are
+ * Data nodes (see :class:`nvidia.dali.pipeline.DataNode`) - represent outputs and inputs of operators; they are
```

Comment:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax
```suggestion
 * Data nodes (see :class:`~nvidia.dali.pipeline.DataNode`) - represent outputs and inputs of operators; they are
```
That should make the link render as `DataNode` instead of the full module path.